docs/changelog.md

```diff
@@ -5,12 +5,30 @@ toc:
 
 # Redocly CLI changelog
 
+## 1.0.0-beta.130 (2023-07-06)
+
+### Features
+
+- Added new rule `component-name-unique` to check for unique component names (kudos to @pboos 🎉).
+- Added docker container to GitHub packages.
+
+### Fixes
+
+- Fixed an issue when the `--files` option of the `push` command did not upload extra files.
+- Fixed wrong schema type validation.
+- Fixed issue with broken styles in `build-docs` command.
```

Comment:
Was this the additional peer dependencies change? I think it might be worth saying something like "Added peer dependencies to fix an issue with styles seen in the `build-docs` command.

Reply:
```suggestion
- Added peer dependencies to fix an issue with styles seen in the build-docs command.
```